app/views/assignments/_peer_submissions_table.js.jsx.erb

```diff
@@ -0,0 +1,151 @@
+<script type='text/jsx'>
+  /** @jsx React.DOM */
+
+  var PeerSubmissionsTable = React.createClass({
+    getDefaultProps: function() {
+      // Defines the columns used for the table and whether they
+      // are sortable searchable. The default initially sorted
+      // column is the first sortable column in the array.
+      return {
+        columns: [
+          {
+            id: 'group_name',
+            content: '<%= j raw I18n.t(:'browse_submissions.group_name') %>',
```

Comment:
Use `t` instead of `I18n.t`. I don't want to have to repeat myself every time you do this.